source/badjs.js

```diff
@@ -0,0 +1,4 @@
+let bobby = fetch(
+  'https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group27/cse110-fa21-group27/pullRequest?prid=8413591)
```suggestion
  "https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging"
```